tests/s3/test_object_workflow_operations.py

```diff
@@ -107,6 +111,43 @@ def create_bucket_put_objects(self, bucket_name, object_count):
 
         return obj_list
 
+    def create_bucket_put_get_object(self, bucket_name, obj_key_lst, file_path, mb_count):
```

Comment:
We can consolidate both create_bucket_put_objects and this one into a single method and rename it to create_bucket_put_get_objects. The method can check if object_count has been passed and subsequently generate the object_list. If object_list is directly passed this step (i.e. object list generation) can be skipped. The method would need to return the object list so that older test cases don't break.

This should also be moved to s3_common_test_lib so that other tests can re-use this going forward. When moving the method to the common lib, care should be taken to accept an additional argument for the S3 test lib object.